keras/backend/tensorflow_backend.py

```diff
@@ -2313,6 +2314,10 @@ def __init__(self, inputs, outputs, updates=None, name=None, **session_kwargs):
                     updates_ops.append(update)
             self.updates_op = tf.group(*updates_ops)
         self.name = name
+        self.fetches = session_kwargs.pop('fetches', [])
+        if not isinstance(self.fetches, list):
```

Comment:
I believe it could be any iterable, so probably better to check for an iterable and cast it as list.